sound/soc/sof/ops.h

```diff
@@ -555,6 +555,14 @@ snd_sof_set_mach_params(struct snd_soc_acpi_mach *mach,
 		sof_ops(sdev)->set_mach_params(mach, sdev);
 }
 
+static inline bool
+snd_sof_is_dai_supported(struct snd_sof_dev *sdev, u32 dai_type, bool dspless_mode)
```

Comment:
From the commit message it is not obvious why the is_dai_supported question is coupled with DSPless mode.

Reply:
fixed now